workspace.bzl

```diff
@@ -8,14 +8,14 @@ def play_routes_compiler_cli_repositories():
     maven_install(
         name = "play_routes_compiler_cli_maven",
         artifacts = [
-            "com.github.scopt:scopt_2.12:3.7.0",
-            "com.typesafe.play:routes-compiler_2.12:2.7.4",
-            "org.scala-sbt:zinc_2.12:1.2.1",
+            "com.github.scopt:scopt_2.13:3.7.1",
+            "com.typesafe.play:routes-compiler_2.13:2.8.2",
+            "org.scala-sbt:zinc_2.13:1.4.0-M6",
```

Comment:
Hmm this is a milestone release; are we okay with using that? It might be less stable